Source/WebCore/svg/svgtags.in

```diff
@@ -8,7 +8,7 @@ altGlyph JSInterfaceName=SVGElement
 altGlyphDef JSInterfaceName=SVGElement
 altGlyphItem JSInterfaceName=SVGElement
 animate
-animateColor
+animateColor interfaceName=SVGUnknownElement, JSInterfaceName=SVGElement, noConstructor
```

Comment:
Can we remove this?

Reply:
We can't since HTML Parsing rules needed to kept intact. I removed in first PR and it failed bunch of WPT tests in relation to HTML5 Parser.

Reply:
@annevk Is this true?

Reply:
Just for reference - https://ews-build.webkit.org/#/builders/1/builds/42456 (If we look into failures here).

Reply:
So the only thing that changes if we remove this is the casing which goes from `animateColor` to `animatecolor`. Seems removable to me, maybe @annevk has guidance on what to do with the tests.

Reply:
I do wonder if we need to set all these interface name thingies. Does anything break if you just leave it as "animateColor"?

Reply:
@annevk - It leads to build error:
```
/Users/ahmadsaleem/Documents/GitHub-Webkit-Ahmad-Fork/Untitled/WebKitBuild/Release/DerivedSources/WebCore/SVGElementFactory.cpp:40:10: fatal error:
      'SVGAnimateColorElement.h' file not found
#include "SVGAnimateColorElement.h"
         ^~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
```

Reply:
Since it leads to compiler issue, I am considering this as 'RESOLVED' unless if in future, we update HTML5 Parser tests and we can remove it completely.